microprofile.jdt/com.redhat.microprofile.jdt.core/src/main/java/com/redhat/microprofile/commons/ProjectLabelInfo.java

```diff
@@ -0,0 +1,57 @@
+/*******************************************************************************
+* Copyright (c) 2020 Red Hat Inc. and others.
+* All rights reserved. This program and the accompanying materials
+* which accompanies this distribution, and is available at
+* http://www.eclipse.org/legal/epl-v20.html
+*
+* Contributors:
+*     Red Hat Inc. - initial API and implementation
+*******************************************************************************/
+package com.redhat.microprofile.commons;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * Project type results providing a mapping from a project to its type
+ *
+ */
+public class ProjectLabelInfo {
+	private List<ProjectLabelInfoEntry> projectLabelEntries;
```

Comment:
private final List<ProjectLabelInfoEntry> projectLabelEntries;

Reply:
Just for clarification, do we want `List` instead of `List<ProjectLabelInfoEntry>`?